public/app/features/alerting/unified/api/prometheusApi.ts

```diff
@@ -54,12 +58,12 @@ export const prometheusApi = alertingApi.injectEndpoints({
       },
     }),
     getGrafanaGroups: build.query<PromRulesResponse<GrafanaPromRuleGroupDTO>, GrafanaPromRulesOptions>({
-      query: ({ namespace, groupName, ruleName, groupLimit, excludeAlerts, groupNextToken }) => ({
+      query: ({ folderUid, groupName, ruleName, groupLimit, excludeAlerts, groupNextToken }) => ({
         url: `api/prometheus/grafana/api/v1/rules`,
         params: {
-          'file[]': namespace,
-          'group[]': groupName,
-          'rule[]': ruleName,
+          folder_uid: folderUid,
```

Comment:
Have we already merged some backend changes to support these new query params?

Reply:
It's quite old actually https://github.com/grafana/grafana/pull/88295